flows/actions/call_classifier.go

```diff
@@ -94,12 +88,12 @@ func (a *CallClassifierAction) classify(run flows.FlowRun, step flows.Step, inpu
 		return nil, false, errors.Errorf("missing %s", a.Classifier.String())
 	}
 
-	nluSvc := run.Session().Engine().Services().NLU(run.Session(), classifier)
-	if nluSvc == nil {
-		return nil, false, errors.Errorf("no NLU service available for %s", a.Classifier)
+	svc := run.Session().Engine().Services().NLU(run.Session(), classifier)
+	if svc == nil {
+		return nil, false, errors.Errorf("no classifier service available for %s", a.Classifier)
```

Comment:
Not really part of this PR but really feels to me that the factory method should return an error so we have some idea as to why nil is being returned here. (IE, missing config values vs it doesn't exist)

Reply:
Let me merge this and take a look at that